docs/how/kafka-config.md

```diff
@@ -19,3 +19,17 @@ DataHub components that connect to Kafka are currently:
 - mce-consumer-job
 - mae-consumer-job
 - gms
+- Various ingestion example apps
+
+## Configuring Topic Names
+
+By default, ingestion relies upon the `MetadataChangeEvent_v4`, `MetadataAuditEvent_v4`, and `FailedMetadataChangeEvent` kafka topics by default for
+[metadata events](https://github.com/linkedin/datahub/blob/master/docs/what/mxe.md.
+
+We've included environment variables to customize the name each of these topics, if your company or organization has naming rules for your topics.
+
+- `METADATA_CHANGE_EVENT_NAME`: The name of the metadata change event topic.
```

Comment:
Would it be a good idea to specify these in all the docker environment using the default values?

Reply:
Good idea, but we should see if we can autogenerate from spring somehow. Manually keeping track is going to be a pain.

I also think it would be a good idea to list this here anyway. Multiple places is fine.